ruby.c

```diff
@@ -241,6 +254,7 @@ usage(const char *name, int help)
     };
     static const struct message help_msg[] = {
 	M("--copyright",                   "", "print the copyright"),
+	M("--dump=argument[,...]",	   "", "argument is such as insns. see --help=dump for more details"),
```

Comment:
You could list some of the valid arguments like so:
`M("--dump={insns|parsetree|...}[,...]", "", "dump debug information; see ..."),`

Reply:
thanks, fixed so

Reply:
Also, changed

```
  --enable=feature[,...], --disable=feature[,...]
                  enable or disable features
...
Features:
  gems            rubygems (default: enabled)
  did_you_mean    did_you_mean (default: enabled)
  rubyopt         RUBYOPT environment variable (default: enabled)
  frozen-string-literal
                  freeze all string literals (default: disabled)
```

to

```
  --enable={gems|did_you_mean|...}[,...], --disable={gems|did_you_mean|...}[,...]
                  enable or disable features shown below
...

...
Features:
  gems            rubygems (default: enabled)
  did_you_mean    did_you_mean (default: enabled)
  rubyopt         RUBYOPT environment variable (default: enabled)
  frozen-string-literal
                  freeze all string literals (default: disabled)
```